### PR TITLE
chore(release): release 18.0.0

### PR DESCRIPTION
**Pull request by Laffery**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,8 +57,8 @@
     "@angular/router": "^18.0.1",
     "@angular/service-worker": "^18.0.1",
     "@angular/ssr": "^18.0.2",
-    "@commitlint/cli": "^19.0.0",
-    "@commitlint/config-angular": "^19.0.0",
+    "@commitlint/cli": "^13.0.0",
+    "@commitlint/config-angular": "^13.0.0",
```
Comment (Laffery) [inline package.json:61]: Due to dependency `conventional-changelog-angular` is v7, which is conflict with dependency of `conventional-changelog-cli`, I failed to generate changelog.
Here I revert the version to preview v13, and I'll solve it later

File: docs/changelog.zh-CN.md [hunk 2]
```diff
@@ -13,6 +13,40 @@ timeline: true
...
+
+* **collapse:** `nzExpandIconPosition` 类型从 `left` | `right` 变更为 `start` | `end` ([#8561](https://github.com/NG-ZORRO/ng-zorro-antd/issues/8561)) ([3ad5674](https://github.com/NG-ZORRO/ng-zorro-antd/commit/3ad56749b0c8222b37444f27f81942fba4bc53e3))
+* Less 中不再使用 inline JavaScript ([#8552](https://github.com/NG-ZORRO/ng-zorro-antd/issues/8552)) ([7e873c8](https://github.com/NG-ZORRO/ng-zorro-antd/commit/7e873c863a1c8e9c053f64aca86bf9c7c9a11a21))
+
+使用内置 Less 函数时不再需要用 ~\`\` 包裹。例如：
+
+```diff
+- color(~`colorPalette('@{primary-color}', 5)`)
++ color(colorPalette('@{primary-color}', 5))
+```
```
Comment (HyperLife1119) [inline docs/changelog.zh-CN.md:30]:
```suggestion
使用 Antd 提供的内置 Less 函数时不再需要用 ~\`\` 包裹，例如：

\```diff
- color(~`colorPalette('@{primary-color}', 5)`)
+ color(colorPalette('@{primary-color}', 5))
\```

受影响的函数包括：colorEasing，colorPalette，tinycolor
```

Comment (Laffery) [reply docs/changelog.zh-CN.md]: updated

File: components/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (HyperLife1119) [inline components/package.json:1]: We also need to update the dependencies and peerDependencies in this file :)

Comment (Laffery) [reply components/package.json:1]: you are right 👍

**Review by HyperLife1119 — APPROVED** [review]
LGTM

**Review by OriginRing — APPROVED** [review]

**Comment by OriginRing** [comment]
LGTM

**Pull request merged**